.travis.yml

```diff
@@ -16,20 +16,6 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
```

Comment:
please focus on psalm integration and do not drop old PHP version.

Reply:
yes, i'm turn back php support <7.1 .travis.yaml